Seldon added to upstream operators

axsaucedo opened the pull request:
Following up from the addition of the seldon operator into the community-operators (through #447) this PR contains the operator submission for the community-operators, and should be enough to close #393.

axsaucedo commented:
@aneeshkp @tmckayus here is the PR with the updated CSV for upstream operators. Not sure why there is the issue `Pod seldon-init-secret listed in CSV was not found in namespace operator` - seems the pod that creates the secret is not working?

tmckayus commented:
interesting @axsaucedo I'll see what I can figure out

tmckayus commented:
/retest

aneeshkp commented:
/test CVP

aneeshkp commented:
@axsaucedo Can you change minKubeVersion: 1.13.0 to minKubeVersion: 1.12.0
CI is not been updated with   kubeversion >1.12.0, so the validation fails. This change is to make the CI pass the validation.

aneeshkp commented:
/test CVP

axsaucedo commented:
I have updated the minKubeVersion @aneeshkp @tmckayus 👍

aneeshkp commented:
@axsaucedo , There is still a mention of secrete requirement in the description

axsaucedo commented:
@aneeshkp thanks for the heads up, I have now copied it to the same contents to the community-operators file but with the minKubeVersion set to `1.12.0`

aneeshkp commented:
@axsaucedo , you mean to upstream-community-operators right?. I don't see any git changes.

axsaucedo commented:
@aneeshkp strange, I had added a commit but it was indeed not appearing. I have added a new commit with just a new line and it seems the previous commit is now visible before your comment. Regardless, it's now updated with the latest, and doesn't include the mentions about the secret.

aneeshkp commented:
@axsaucedo  I see it now. Thanks

aneeshkp commented:
/test CVP
